test/omise/test_account.rb

```diff
@@ -11,10 +11,10 @@ def test_that_we_can_retrieve_the_account
   end
 
   def test_that_we_can_reload_the_account
-    @account.attributes.taint
+    @account.attributes.frozen?
```

Comment:
@ajzkk this seems useless https://ruby-doc.org/3.1.2/Kernel.html#method-i-frozen-3F

Did you mean `.freeze`?

Reply:
Yes. It should be `.freeze`. I have updated.